addons/web/static/src/js/core/misc.js

```diff
@@ -51,6 +51,25 @@ if ($.blockUI) {
     $.blockUI.defaults.css["background-color"] = '';
 }
 
+var elementWithAccessKey = [];
```

Comment:
this might represent a memory leak: you are keeping a reference to all elements with accesskeys alive forever. I would not store the elements outside a the function, and would instead in the unblockAccessKeys do a document.querySelectorAll("[data-accesskey]") to restore the access keys